### DISCUSSION TITLE
Dev Environment on MAC OSX not working locally

### DISCUSSION THREAD

**Discussion by accountunavaible** [Budibase/budibase#9445] [Contributing]
hi guys, I am new to budibase, I wanted to try it locally, I followed the docs from below link
[doc for mac os](https://github.com/Budibase/budibase/blob/develop/docs/DEV-SETUP-MACOSX.md)

I did the exactly the same thing, also matched the exact version for node, yarn, docker, docker-compose and lerna, also uncomment the linux/amd64, and added eval into zsh.

but when I run commend in terminal yarn setup: 

it will pops error like : " Lerna ERR yarn install mutex network:42424 non interactive....", then I searched up from google, I noticed some developers were facing same issue in the budibase, then I followed the solution here [solution](https://github.com/Budibase/budibase/discussions/5560) but it still did not work. 

but with budibase cli, I am able to run the project, but I only get .env and docker-compose.yaml, where can I link the code in the docker, I tried app-service, but it does not have it. Thank you guys!!!

**Comment by melohagan** [comment]
Hey @accountunavaible 

Here's some things to try.

1. Make sure no existing Budibase containers are running, or any other containers that could conflict on the ports
2. Pull down the latest `develop` branch
3. `yarn`
4. `yarn setup`

If there's still issues run `yarn nuke`, and then try the `yarn setup` again.

**Reply by accountunavaible** [reply]
Thank you very much for you help, I solved the problem by changing all the yarn to npm

**Reply by melohagan** [reply]
Oh! Out of curiosity what versions of npm and yarn are you running?

**Reply by accountunavaible** [reply]
I am using npm version 9.1.3, yarn is 1.22.19

**Reply by melohagan** [reply]
That should be grand, don't see any reason why yarn wouldn't work for you!

**Reply by accountunavaible** [reply]
yeah, I was thinking the same thing
